src/Controls/samples/Controls.Sample.Droid/Maui.Controls.Sample.Droid-net6.csproj

```diff
@@ -1,11 +1,12 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk.Razor">
```

Comment:
@jonathanpeppers how do we make this not needed, and instead import from a property?

Reply:
So if we followed the pattern of what `$(UseMaui)` will eventually be, maybe we should come up with a new property like `$(UseRazor)`?

I think we could do something like:

```xml
<PropertyGroup>
  <_RazorSdkImportsMicrosoftNetSdk  Condition=" '$(UseRazor)' == 'true' ">false</_RazorSdkImportsMicrosoftNetSdk>
</PropertyGroup>
<Import Condition=" '$(UseRazor)' == 'true' " Sdk="Microsoft.NET.Sdk.Razor" Project="Sdk.props" />
```
And put this in `Microsoft.Maui.Controls.props` ^^

We'd also need to import in `Microsoft.Maui.Controls.targets`:

```xml
<Import Condition=" '$(UseRazor)' == 'true' " Sdk="Microsoft.NET.Sdk.Razor" Project="Sdk.targets" />
```

Then any project that has `Microsoft.NET.Sdk.Razor` would define `<UseRazor>true</UseRazor>` instead.